assets/scss/hero.scss

```diff
@@ -82,3 +82,75 @@
     }
   }
 }
+
+.exhibition-page {
+  h1 {
+    font-size: 2.375rem;
```

Comment:
To keep it consistent with the other h1s, could you remove the font-size? 
I have verified this with the design team.